packages/flutter/test/material/chip_test.dart

```diff
@@ -1912,6 +1912,36 @@ void main() {
     await gesture.removePointer();
   });
 
+  testWidgets('Chips with background color should have delete icon ripple and splash in the foreground', (WidgetTester tester) async {
+    // Regression test for https://github.com/flutter/flutter/issues/41461
+    final Key deleteKey = UniqueKey();
+    await tester.pumpWidget(
+      _wrapForChip(
+        child: Chip(
+          onDeleted: () { },
+          backgroundColor: Colors.pink,
+          label: const Text('Chip with delete icon'),
+          deleteIcon: Icon(Icons.cancel, key: deleteKey),
```

Comment:
Probably don't need the deleteKey; could just use find.byType(Icon)

Reply:
Done!